modules/nodes-cma-autoscaling-custom-trigger-kafka.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * nodes/cma/nodes-cma-autoscaling-custom-trigger.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-cma-autoscaling-custom-trigger-kafka_{context}"]
+= Understanding the Kafka trigger
+
+You can scale pods based on an Apache Kafka topic or other services that support the Kafka protocol. The custom metrics autoscaler does not scale higher than the number of Kafka partitions, unless you set the `allowIdleConsumers` parameter to `true` in the scaled object or scaled job.
+
+:FeatureName: Autoscaling based on Apache Kafka metrics
+include::snippets/technology-preview.adoc[leveloffset=+0]
+
+[NOTE]
+====
+If the number of consumer groups exceeds the number of partitions in a topic, the extra consumer groups sit idle.
+
+To avoid this, by default the number of replicas does not exceed:
```

Comment:
```suggestion
To avoid this, by default the number of replicas does not exceed the following items:
```

Reply:
^ Maybe remove spacing between this and the previous sentence so it's more clear what "this" in "to avoid this" is referring to.